health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignType.js

```
@@ -156,7 +158,44 @@
+              label={t("PREVIEW")}
+              onClick={() => {
+                setShowPopUp1(true);
+              }}
+            /> */}
           </Card>
+          {showPopUp1 && (
+            <PopUp
+              className={"custom-pop-up"}
+              type={"default"}
+              heading={t("CHECKLIST_PREVIEW")}
+              children={[]}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)